Ollivanders/src/net/pottercraft/Ollivanders2/StationarySpell/COLLOPORTUS.java

```diff
@@ -17,9 +20,38 @@ public COLLOPORTUS (Player player, Location location, StationarySpells name, Int
       super(player, location, name, radius, duration);
    }
 
+   public COLLOPORTUS (Player player, Location location, StationarySpells name, Integer radius, Integer duration,
+                       Map<String, String> spellData, Ollivanders2 plugin)
+   {
+      super(player, location, name, radius, duration);
+
+      deserializeSpellData(spellData, plugin);
+   }
+
    public void checkEffect (Ollivanders2 p)
    {
       if (duration < 1)
          kill();
    }
+
+   /**
+    * Serialize all data specific to this spell so it can be saved.
+    *
+    * @param p unused for this spell
+    * @return a map of the serialized data
+    */
+   @Override
+   public Map<String, String> serializeSpellData (Ollivanders2 p)
+   {
+      return new HashMap<>();
```

Comment:
why do the serialize and deserialize do nothing?

Reply:
I have not written the deserialize yet. That is the next task. Step 1 is make sure we're writing the files correctly and nothing blows up.